test/Microsoft.AspNet.Mvc.Core.Test/OptionDescriptors/InputFormatterDescriptorExtensionTest.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Collections.Generic;
+using System.Linq;
```

Comment:
Left over from `RemoveTypesOf` test which I removed after deleting the method from `InputFormatterDescriptorExtensions`. Will remove.